### PR TITLE
GHA-86 PREQ: Do not assign tickets

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
[GHA-86](https://sonarsource.atlassian.net/browse/GHA-86)

[GHA-86]: https://sonarsource.atlassian.net/browse/GHA-86?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: PullRequestCreated/PullRequestCreated.test.ts [hunk 1]
```diff
@@ -204,7 +204,6 @@ describe('PullRequestCreated', () => {
       "Invoked jira.addIssueRemoteLink('PREQ-4242'', 'https://github.com/test-owner/test-repo/pull/42', null)",
       "Invoked jira.moveIssue('PREQ-4242', 'Commit', null)",
       "Invoked jira.moveIssue('PREQ-4242', 'Start', null)",
-      "Invoked jira.assignIssueToAccount('PREQ-4242', '1234-account')",
```
Comment (mary-georgiou-sonarsource) [inline PullRequestCreated/PullRequestCreated.test.ts:207]: maybe it's better to add a dedicetated test "Is PREQ assignee is null"?

**Review by mary-georgiou-sonarsource — COMMENTED** [review]
Comment about addind a dedicated test

**Review by mary-georgiou-sonarsource — APPROVED** [review]
👁️ ok
